### PR TITLE
FFTW++ integration - v1

### PR DESCRIPTION
**Pull request by af-ayala**
Integration of library FFTW++.
A CPU backend created on top of FFTW.

Initial template.

### CODE REVIEW

File: include/fiber_backends.h [hunk 1]
```diff
@@ -59,6 +61,7 @@ const struct fiber_map_backend_d2z fiber_execute_d2z[] = {
     { "nb3dfft",    &compute_d2z_nb3dfft   },
     { "fftw",       &compute_d2z_fftw      },
     { "fftadvmpi",  &compute_d2z_fftadvmpi }
```
Comment (stomov) [inline include/fiber_backends.h]: Looks like some "," are missing here, above and below.

Comment (af-ayala) [reply include/fiber_backends.h]: Thanks, Stan,  fixed it!

File: include/fiber_backend_fftwpp.h [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    MPI_Barrier(comm);
+    timer[1] = +MPI_Wtime();
+
+}
+
+
+#else
+
+int init_fftwpp(int option)
+{}
```
Comment (luszczek) [inline include/fiber_backend_fftwpp.h]: Add `retrun 0;` to avoid compiler warning/error.

File: include/fiber_backends.h [hunk 3]
```diff
@@ -80,6 +83,7 @@ const struct fiber_map_backend_z2d fiber_execute_z2d[] = {
     { "nb3dfft",    &compute_z2d_nb3dfft   },
     { "fftw",       &compute_z2d_fftw      },
     { "fftadvmpi",  &compute_z2d_fftadvmpi }
+    { "fftwpp",     &compute_z2d_fftwpp }
```
Comment (luszczek) [inline include/fiber_backends.h:86]: Same thing as above: add comma to separate backends

File: include/fiber_backends.h [hunk 4]
```diff
@@ -101,6 +105,7 @@ const struct fiber_map_backend_z2z fiber_execute_z2z[] = {
     { "nb3dfft",    &compute_z2z_nb3dfft   },
     { "fftw",       &compute_z2z_fftw      },
     { "fftadvmpi",  &compute_z2z_fftadvmpi }
```
Comment (luszczek) [inline include/fiber_backends.h]: Add a comma here too.

**Pull request merged**